src/elements/card/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.14.5",
   "license": "MIT",
   "main": "lib/index.js",
+  "ts:main": "src/index.tsx",
```

Comment:
what is this for?

Reply:
without it, i could not use `@uswitch/trustyle.card` as a dependency of the `@uswitch/trustyle.link-list`

I tried simply installing `card` as an `npm` dep inside the `link-list` package, but that didn't work. I think it's due to how lerna bootstrap works - when i looked at what it was doing for another dep, `icon`, it was simlinking the icon folder into link-list's node_modules - but when i did the same for `card`, it didn't work, until i added the `ts:main` value (after manually editing `link-lists`s package.json)

This particular line isn't required for this pr, as I removed the `card` dep and added the styling directly to the `link-list` - should i remove it ? I left it in so it could be used in this manner in the future, but that might not be necessary.